pkg/cmd/types.go

```diff
@@ -99,15 +113,15 @@ func (i *ImageInspectorOptions) Validate() error {
 		}
 	}
 	if len(i.ScanType) > 0 {
-		var found bool = false
-		for _, opt := range ScanOptions {
-			if i.ScanType == opt {
-				found = true
-				break
-			}
+		if !util.StringInList(i.ScanType, ScanOptions) {
+			return fmt.Errorf("%s is not one of the available scan-types which are %v",
+				i.ScanType, ScanOptions)
 		}
-		if !found {
-			return fmt.Errorf("%s is not one of the available scan-types which are %v", i.ScanType, ScanOptions)
+	}
+	if len(i.PullPolicy) > 0 {
```

Comment:
I don't think you need the length check here.  We provide a default and if the user sets it to empty we want the error to show.